### PR TITLE
Release 11.0.0

### PR DESCRIPTION
**Pull request by hobbitronics**
followed https://storybook.js.org/docs/migration-guide
ran npx storybook@7 upgrade

### Added
- add some dependencies when running the script (react, react-dom, etc)
- add some alternative build scripts when running the script
- add `autoDocs: true` to get Docs working in Stories
- add storybook/svelte-vite to simplify build config and improve performance

### Removed
- remove webpack and config, addon-postcss, css-loader, style-loader and a few more during migration

### Changed (BREAKING)
- update svelte 3 to 4 (BREAKING CHANGE)
- updated storybook 6 to 7
- updated other dependencies including sass
- update actions

### Fixed
- link.href typo
- Fix failed GitHub pages deployment

BREAKING CHANGE: consumers now need to use svelte 4 to not get npm error when installing

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -1,44 +1,26 @@
...
-
-module.exports = {
+/** @type { import('@storybook/svelte-vite').StorybookConfig } */
+const config = {
   "stories": [
     "../**/*.stories.mdx",
     "../**/*.stories.@(js|jsx|ts|tsx|svelte)"
   ],
-  "addons": [
+  addons: [
```
Comment (briskt) [inline .storybook/main.js:7]: Why is this one unquoted but the others still have quotes?

File: stories/Alert.stories.svelte [hunk 2]
```diff
@@ -11,7 +11,7 @@ const args = {
 }
 </script>
 
-<Meta title="Atoms/Dialog.Alert" component={Dialog.Alert} />
+<Meta title="Atoms/Dialog.Alert" tags={['autodocs']} component={Dialog.Alert} />
```
Comment (briskt) [inline stories/Alert.stories.svelte:14]: Is there a way to make this a default so it works on all stories without specifying this tag?

**Review by briskt — APPROVED** [review]

File: stories/Alert.stories.svelte (hunk 2 above)
Comment (hobbitronics) [reply stories/Alert.stories.svelte:14]: Yes there is, thanks for asking.

File: .storybook/main.js (hunk 1 above)
Comment (hobbitronics) [reply .storybook/main.js:7]: it only needs the quotes when there are special characters in the strings, so I will remove them.

**Pull request merged**